vite.config.js

```diff
@@ -17,6 +17,7 @@ export default defineConfig({
 			// into your library
 			external: ['@vueuse/core', 'vue'],
 			output: {
+				exports: 'named',
```

Comment:
See https://rollupjs.org/configuration-options/#output-exports

Reply:
Because of this code of `index.js`:
```js
export default function (options) {
    Options.setProviderOptions(options);
}
```

There are default exports and named exports and we have to define the output exports as named to prevent errors.